packages/dds/shared-object-base/src/TelemetryProperties.ts

```diff
@@ -0,0 +1,13 @@
+/*!
+ * Copyright (c) Microsoft Corporation and contributors. All rights reserved.
+ * Licensed under the MIT License.
+ */
+
+/**
+ * Telemetry properties to include for the {@link TelemetryEventBatcher} logger.
+ * The set of properties must be the same for all calls to the `measure` function.
+ * @internal
+ */
```

Comment:
My take for creating a new file is that I wanted this file to be a compilation of all the to-be created custom telemetry properties in the future. But I can go ahead and simply include this interface inside the `SharedObjet.ts`.

Reply:
Yeah, I get that line of thought. I would still push back, because the telemetry properties in each place that needs them are going to be pretty specific to that place, not really shareable (technically possible? yes, but my gut feeling says it won't be recommendable even when possible), so I think they make more sense accompanying the specific JS module (file) that uses them.